go/tasks/plugins/presto/test_helpers.go

```diff
@@ -0,0 +1,112 @@
+package presto
```

Comment:
Can you rename the file to `helpers_test.go` or add the annotation to only build it for Test? I know go is pretty good at trimming what's not used but why put strain on the compiler if we don't need to..